### ISSUE TITLE
implement fluent zooming feature

### ISSUE THREAD

**Issue by maiklos** [CompendiumNG/CompendiumNG#14]
allow to zoom map fluently by holding control and rotating the wheel-mouse up or down.

allow for more fine zoom level selection and to select the zoom-level easily

**Comment by SebastianBoell** [comment]
This is an absolutely essential improvement for Compendium to bring it up to scratch with modern canvas based application interfaces. I would like to add:
a) The possibility to click and drag (shall we open a new issue for this)
b) This will free the F2 button from the zooming function to be allocated to the 'default' function of renaming the item that is currently selected (as it the function of F2 in many other applications)

**Comment by maiklos** [comment]
implemented and merged into M1 in commit: a5b3047

```
add more finegrained map zooming
 - associate zooming of a map with Ctrl-wheel
 - synchronize displayed zoom level when zooming is done via mouse
 - make zooming level 10-100% by 10, 100-200% by 25
 - remove <F2> association
 - add event logging via AspectJ in debug level
```

**Comment by SebastianBoell** [comment]
Great work! This is really much better than before. I guess for a future release zoom should home in on the mouse position this way zoming in and out will enable easy navigation on big maps.
